### ISSUE TITLE
Sereal-Path-0.14: Devel::CheckLib not required/included

### ISSUE THREAD

**Issue by eserte** [Sereal/Sereal#160]
While trying to configure the distribution:
```
Output from '/opt/perl-5.24.1/bin/perl Makefile.PL':

Checking if your kit is complete...
Looks good
Warning: prerequisite Sereal::Decoder 3.00 not found.
Warning: prerequisite Sereal::Encoder 3.00 not found.
ERROR from evaluation of /home/eserte/.cpan/build/2017020718/Sereal-Path-0.014-9Yhg3M/Iterator/Makefile.PL: Can't locate Devel/CheckLib.pm in @INC (you may need to install the Devel::CheckLib module) (@INC contains: .. ../inc /home/eserte/.cpan/build/2017020718/Sereal-Path-0.014-9Yhg3M /home/eserte/.cpan/build/2017020718/Sereal-Path-0.014-9Yhg3M/inc /var/tmp/cpansmoker-1000/2017020718/CPAN-Reporter-lib-SS9a /opt/perl-5.24.1/lib/site_perl/5.24.1/x86_64-linux /opt/perl-5.24.1/lib/site_perl/5.24.1 /opt/perl-5.24.1/lib/5.24.1/x86_64-linux /opt/perl-5.24.1/lib/5.24.1 /home/eserte/.cpan/build/2017020718/Sereal-Path-0.014-9Yhg3M .) at inc/Sereal/BuildTools.pm line 81.
```

Probably Devel::CheckLib should be added to configure_requires, or included into inc/.

**Comment by ikruglov** [comment]
Thank you for reporting. I will fix it.

**Comment by ikruglov** [comment]
included into 4.001_001
